Add Installation and Getting Started docs (#120)
Oneflow-Inc/libai: docs/source/tutorials/Getting Started.md

# Getting Started

> This page provides basic tutorials about the usage of LiBai:

Reviewer: ```suggestion
This page guides you through a basic way to get started with LiBai:
```
参考的这句话“This page guides you through the quickest way to get started with CockroachDB”

Author: wow~非常nice

- [Train VisionTransformer on ImageNet Dataset](#train-visiontransformer-on-imagenet-dataset)
  - [Data preparation](#data-preparation)
  - [Train vit model from scratch](#train-vit-model-from-scratch)

## Train VisionTransformer on ImageNet dataset

> ### Data preparation

Reviewer: ```suggestion
### Prepare the Data
```

For ImageNet, we use standard ImageNet dataset, you can download it from http://image-net.org/.
- For standard folder dataset, move validation images to labeled sub-folders. The file structure should be like:
```bash
$ tree data
imagenet
├── train
│   ├── class1
│   │   ├── img1.jpeg
│   │   ├── img2.jpeg
│   │   └── ...
│   ├── class2
│   │   ├── img3.jpeg
│   │   └── ...
│   └── ...
└── val
    ├── class1
    │   ├── img4.jpeg
    │   ├── img5.jpeg
    │   └── ...
    ├── class2
    │   ├── img6.jpeg
    │   └── ...
    └── ...

```
### Train vit model from scratch

> - update the data path in [vit_imagenet](https://github.com/Oneflow-Inc/libai/blob/main/configs/vit_imagenet.py) config file

Reviewer: ```suggestion
- Update the data path in [vit_imagenet](https://github.com/Oneflow-Inc/libai/blob/main/configs/vit_imagenet.py) config file
```

```python
# Refine data path to imagenet data folder
dataloader.train.dataset[0].root = "/path/to/imagenet"
dataloader.test[0].dataset.root = "/path/to/imagenet"
```

> - To train `vit_tiny_patch16_224` model on ImageNet on a single node with 8 gpus for 300 epochs, run:

Reviewer: ```suggestion
- To train `vit_tiny_patch16_224` model on ImageNet on a single node with 8 GPUs for 300 epochs, run:
```

```bash
bash tools/train.sh configs/vit_imagenet.py 8
```

> - In LiBai we set the default vit model to `vit_tiny_patch16_224`. To train other vit model you can update the [vit_imagenet](https://github.com/Oneflow-Inc/libai/blob/main/configs/vit_imagenet.py) config file by importing the other vit model in the config file as follows:

Reviewer 2: ```suggestion
- In LiBai we set the default vit model to `vit_tiny_patch16_224`. To train other vit models you can update the [vit_imagenet](https://github.com/Oneflow-Inc/libai/blob/main/configs/vit_imagenet.py) config file by importing the other vit models in the config file as follows:
```

Reviewer: In LiBai we set the default vit model to `vit_tiny_patch16_224`. 
这里是指默认的模型是这个吗？如果是，本句可以改成：
The default vit model in LiBai is `vit_tiny_patch16_224`.
